src/unitTypes.js

```diff
@@ -93,7 +93,7 @@ export const minuteSecond = (seconds, stringOpts) => ({
 
 export const hourMinuteSecond = (seconds, stringOpts) => ({
```

Comment:
odd to me to call this `hourMinuteSecond` if we're gonna get back `hh:mm` regardless of what label we ask for. can we either change this to `hourMinute` or implement a `hh:mm:ss` `long` label?

Reply:
@wceichelb agreed, this is confusing. I was reluctant to change the name because it was an `export`, but I just went and audited rwgps-ui and we don't call it directly anywhere.

So I think it's safe to change.